### PR TITLE
Implement ASP.NET Front-end

### LINKED ISSUE #141: Implement ASP.NET Front-end
## About

As part of the overall Miunie plan, ASP.NET should be one of the front ends.

This issue represents implementation of the ASP.NET project into our current solution.

### PR DESCRIPTION
**Pull request by petrspelos**
## Related Issue

Fixes #141 

## Changes
- Create ASP.NET Server project
- Implement SignalR for client update
- Implement basic bot connection similar to ConsoleApp (no Impersonation yet)

## Expected Feedback
* We'll need to verify this works on other machines.
* When it is your first time running, it will most likely ask about adding certificates, yes it all.
* Your browser might also refuse to connect because the certificate is just a dev one, you can ignore that or add an exception for the domain.
* When using registration for the first time, your DB might not be setup and might require migration, this will result in an exception page, but you can click the "Migrate" or "Start Migration" button to fix it. Once you do, it should work fine.
* Logging in is not necessary to use the bot, it is there for potential future features including "Login with Discord".
* This requires you to include "ASP.NET Web Development" in your Visual Studio Installer. Ideally you'd use Visual Studio 2019.

### CODE REVIEW

File: src/Miunie.Server/Views/Shared/_LoginPartial.cshtml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+else
+{
+    <li class="nav-item">
+        <a class="nav-link text-dark" asp-area="Identity" asp-page="/Account/Register">Register</a>
+    </li>
+    <li class="nav-item">
+        <a class="nav-link text-dark" asp-area="Identity" asp-page="/Account/Login">Login</a>
+    </li>
+}
+</ul>
```
Comment (svr333) [inline src/Miunie.Server/Views/Shared/_LoginPartial.cshtml]: Missing newline

File: src/Miunie.Server/libman.json [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    {
+      "library": "@aspnet/signalr@1.1.0",
+      "destination": "wwwroot/lib/signalr/",
+      "files": [
+        "dist/browser/signalr.js",
+        "dist/browser/signalr.min.js"
+      ]
+    }
+  ]
+}
```
Comment (svr333) [inline src/Miunie.Server/libman.json]: Missing newline

File: src/Miunie.Server/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.css [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+}
+
+template {
+  display: none;
+}
+
+[hidden] {
+  display: none !important;
+}
+/*# sourceMappingURL=bootstrap-reboot.css.map */
```
Comment (svr333) [inline src/Miunie.Server/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.css]: missing newline

File: src/Miunie.Server/Views/Shared/_LoginPartial.cshtml (hunk 1 above)
Comment (petrspelos) [reply src/Miunie.Server/Views/Shared/_LoginPartial.cshtml]: this is boilerplate code

File: src/Miunie.Server/libman.json (hunk 2 above)
Comment (petrspelos) [reply src/Miunie.Server/libman.json]: This is boilerplate code

File: src/Miunie.Server/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.css (hunk 3 above)
Comment (petrspelos) [reply src/Miunie.Server/wwwroot/lib/bootstrap/dist/css/bootstrap-reboot.css]: This is boilerplate code.

**Pull request merged**